crmsh/bootstrap.py

```diff
@@ -688,7 +689,7 @@ def start_pacemaker():
             SBDManager.is_delay_start():
         pacemaker_start_msg += "(waiting for sbd {}s)".format(SBDManager.get_suitable_sbd_systemd_timeout())
     with logger_utils.status_long(pacemaker_start_msg):
-        utils.start_service("pacemaker.service", enable=True)
+        utils.start_service("pacemaker.service", enable=True, node_list=node_list)
```

Comment:
It's a bit tricky here when start --all or multiple nodes.

sle15sp3-1:~ # time crm cluster start --all
INFO: Starting pacemaker(waiting for sbd 36s)...done    
<--- This above line is doesn't match the real "time" consumed, and likely confuse users.
       It does not fit status_long() context. Maybe slightly different wording, ie. 
       INFO: Started pacemaker.service (NOTE: it will wait sbd.service in the background for 36s)

INFO: Cluster services started on sle15sp3-1
INFO: Cluster services started on sle15sp3-2
INFO: Cluster services started on sle15sp3-3

real    0m2.912s
user    0m0.455s
sys     0m0.089s

Reply:
```
# time crm cluster start --all
INFO: Starting pacemaker(waiting for sbd 36s)...done
INFO: Cluster services started on 15sp2-1
INFO: Cluster services started on 15sp2-2
INFO: Cluster services started on 15sp2-3

real    0m33.171s
user    0m2.819s
sys     0m0.879s
```
The real time will be slightly less than expected wait time, it's true; since the wait time will be 1.2*2*SBD_WATCHDOG_TIMEOUT

But I never saw this real time: 2.9s ...

Reply:
Yeah, with the latest version. I see the same as you. Good!